nixos/modules/services/cluster/k3s/default.nix

```diff
@@ -372,7 +372,7 @@ in
           (pkgs.dockerTools.pullImage {
             imageName = "docker.io/bitnami/keycloak";
             imageDigest = "sha256:714dfadc66a8e3adea6609bda350345bd3711657b7ef3cf2e8015b526bac2d6b";
-            sha256 = "0imblp0kw9vkcr7sp962jmj20fpmb3hvd3hmf4cs4x04klnq3k90";
+            hash = "sha256-IM2BLZ0EdKIZcRWOtuFY9TogZJXCpKtPZnMnPsGlq0Y=";
```

Comment:
Since this is touching k3s, do you maybe want to test it? @superherointj 
I am not using docker that much lately.

Reply:
Or maybe can you also update `./examples.nix`. Than I have an example for testing.